src/wfdir.c

```diff
@@ -645,7 +645,15 @@ DirWndProc(
       if (j == cItems)
          return -2L;
 
-      return((i + j) % cItems);
+      int pos = (i + j) % cItems;

```

Comment:
Please avoid C99 variable declarations - they have to be changed to support older systems, and it doesn't seem like we gain anything from using them.

Reply:
I have changed this for here, but I very much prefer C99 declaration since expresses kind of 'locality' much more. Anyhow ;-)